helm/chaos-mesh/README.md

```diff
@@ -0,0 +1,138 @@
+| `prometheus.service.type` | Kubernetes Service type | `ClusterIP` |
+| `prometheus.volume.storage` | | `2Gi` |
+| `prometheus.volume.storageClassName` | | `standard` |
+| `preJobs.podAnnotations` | | `{}` |
+| `postJobs.podAnnotations` | | `{}` |
+| `webhook.certManager.enabled` | Setup the webhook using cert-manager | `false` |
+| `webhook.certManager.autoDeleteSecret` | Whether cert-manager automatically removes the secret resource | `false` |
+| `webhook.FailurePolicy` | Defines how unrecognized errors and timeout errors from the admission webhook are handled | `Ignore` |
+
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
+```console
+# helm 2.X
+helm install helm/chaos-mesh --name=chaos-mesh --namespace=chaos-testing --set dashboard.create=true
+# helm 3.X
+helm install chaos-mesh helm/chaos-mesh --namespace=chaos-testing --set dashboard.create=true
+```
+
+The above command enable the Chaos Dashboard.
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```console
+# helm 2.X
+helm install helm/chaos-mesh --name=chaos-mesh --namespace=chaos-testing -f values.yaml
+# helm 3.X
+helm install chaos-mesh helm/chaos-mesh --namespace=chaos-testing -f values.yaml
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+## Configuration and installation details
+
+### Using cert-manager for certificate management
+
+[Cert-manager](https://github.com/jetstack/cert-manager) may be the default in the K8s world for certificate management now.If you want to install Cert-manager using the [Helm](https://helm.sh) package manager, please refer to the [official documents](https://github.com/jetstack/cert-manager/tree/master/deploy/charts/cert-manager).
+
+Example for deploy Cert-manager
+
+```bash
+kubectl create namespace cert-manager
+kubectl apply --validate=false -f https://raw.githubusercontent.com/jetstack/cert-manager/v0.13.1/deploy/manifests/00-crds.yaml
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+helm install cert-manager jetstack/cert-manager --namespace cert-manager --version v0.13.1
+```
+
+In case you want to using Cert-manager for certificate management, you can use the `webhook.certManager.enabled` property.
+
+```yaml
+webhook:
+  certManager:
+    enabled: true
+```
+
+The webhook's cert and the [MutatingAdmissionWebhook](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook)'s `caBundle` property will be managed by the [Certificate](https://cert-manager.io/docs/usage/certificate/) of Cert-manager.
+
+In case your Cert-manager's option `enable-certificate-owner-ref` is true, it means that deleting a certificate resource will also delete its secret. You can set the `webhook.certManager.autoDeleteSecret` property to `true`.
+
+```yaml
+webhook:
+  certManager:
+    enabled: true
+    autoDeleteSecret: true
```

Comment:
`autoDeleteSecret` is misleading, How about naming it `deleteSecret`, If the `deleteSecret` is true, we will create a job to delete the secret, Otherwise, do nothing

Reply:
Good advice. I'm confused about that too.